src/Model/Map.hs

```diff
@@ -28,35 +28,33 @@ initialMap :: Map
 initialMap = foldr (\index acc -> addMapRow index acc) [] [0..(levelRowNumber initialLevel - 1)]
 
 addMapRow :: Int -> Map -> Map
-addMapRow row list = foldr (\index acc -> if even index
-                                          then addBasicMapTile (Solid, floorTexture) row index acc
-                                          else addBasicMapTile (Transparent, backgroundTexture) row index acc)
+addMapRow row list = foldr (\col acc -> if row + col < 30
```

Comment:
Change number 30 to some named constant.

Reply:
why? this function does not  require further extension and this constant is not used anywhere else.